### PR TITLE
clippyfy

### PR DESCRIPTION
**Pull request by pythcoiner**

### CODE REVIEW

File: gui/src/theme/mod.rs [hunk 1]
```diff
@@ -11,6 +11,7 @@ use iced::{
     },
 };
 
+#[allow(unused)]
```
Comment (darosior) [inline gui/src/theme/mod.rs:14]: If you don't use it, why would you keep it?

Comment (pythcoiner) [reply gui/src/theme/mod.rs:14]: to avoid reimplement it in case we want use other kind items, but i can remove all unused  id you prefer

Comment (darosior) [reply gui/src/theme/mod.rs:14]: Yeah, we should not keep dead code laying around.

Comment (pythcoiner) [reply gui/src/theme/mod.rs]: done

**Pull request merged**